catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/security/ShareableMetacardPreIngestPluginTest.java

```diff
@@ -117,10 +117,10 @@ public void testUpdateIgnoreNonWorkspaceMetacards() throws Exception {
 
   @Test
   public void testUpdatePreserveOwner() throws Exception {
-    WorkspacePreIngestPlugin wpip = makePlugin(ImmutableMap.of());
+    ShareableMetacardPreIngestPlugin wpip = makePlugin(ImmutableMap.of());
 
-    WorkspaceMetacardImpl prev = new WorkspaceMetacardImpl("id");
-    WorkspaceMetacardImpl next = WorkspaceMetacardImpl.clone(prev);
+    ShareableMetacardImpl prev = new ShareableMetacardImpl("id");
```

Comment:
is `"id"` `Core.ID`?

Reply:
`MetacardImpl` uses `Metacard.ID` in the chain of calls, but everything else references `Core.ID`.  So if `Metacard.ID` and `Core.ID` ever diverge from one another and `MetacardImpl` gets updated, these tests will still pass.